packages/kbn-config-schema/src/types/stream_type.test.ts

```diff
@@ -57,7 +57,13 @@ test('includes namespace in failure', () => {
 describe('#defaultValue', () => {
   test('returns default when undefined', () => {
     const value = new Stream();
-    expect(schema.stream({ defaultValue: value }).validate(undefined)).toStrictEqual(value);
+    expect(schema.stream({ defaultValue: value }).validate(undefined)).toMatchInlineSnapshot(`
```

Comment:
Should  `Stream` object serialization change, the test fails. Is it possible to compare by reference?

Reply:
If we do, the test fails with this error:

![image](https://user-images.githubusercontent.com/10602/98657687-6983ef80-2342-11eb-9691-940d67f1849b.png)

I did look into the details when I was debugging it, but at this point I've honestly forgot why this difference is there since it's a while back, but it made sense and I concluded this change was the easiest approach to making the test pass.

Do you want me to investigate again?

Reply:
Let's if the test passes with `toBe`. If not, let's merge as-is. We can fix it later.

Reply:
Same error:

![image](https://user-images.githubusercontent.com/10602/98673429-98f12700-2357-11eb-975d-0fcff93d4ae4.png)

I'll leave it as-is for now then.